### PR TITLE
Update receiver READMEs

### PR DESCRIPTION
**Pull request by flands**
- Add pipeline data types
- Add getting started
- Add advanced configuration where applicable
- Ensure consistent

### CODE REVIEW

File: receiver/README.md [hunk 1]
```diff
@@ -46,10 +55,27 @@ receivers:
...
 such as in pipelines. A full name consists of the receiver type, '/' and the
 name appended to the receiver type in the configuration. All receiver full names
-must be unique. For the example above:
-- Receiver 1 has full name `examplereceiver`.
-- Receiver 2 has full name `examplereceiver/settings`.
+must be unique.
+
+For the example above:
+
+- Receiver 1 has full name `exampleexporter`.
```
Comment (tigrannajaryan) [inline receiver/README.md:62]:
```suggestion
- Receiver 1 has full name `examplereceiver`.
```

File: receiver/README.md [hunk 2]
```diff
@@ -46,10 +55,27 @@ receivers:
...
+- Receiver 2 has full name `exampleexporter/settings`.
```
Comment (tigrannajaryan) [inline receiver/README.md:63]:
```suggestion
- Receiver 2 has full name `examplereceiver/settings`.
```

File: receiver/README.md [hunk 3]
```diff
@@ -46,10 +55,27 @@ receivers:
...
 
-All receivers expose a setting to disable it, by default receivers are enabled.
-At least one receiver must be enabled per [pipeline](../docs/pipelines.md) to be a
-valid configuration.
+Exporters are enabled upon being added to a pipeline. For example:
```
Comment (tigrannajaryan) [inline receiver/README.md:65]:
```suggestion
Receivers are enabled upon being added to a pipeline. For example:
```

File: receiver/README.md [hunk 4]
```diff
@@ -46,10 +55,27 @@ receivers:
...
+must be unique.
+
+For the example above:
+
+- Receiver 1 has full name `exampleexporter`.
+- Receiver 2 has full name `exampleexporter/settings`.
 
-All receivers expose a setting to disable it, by default receivers are enabled.
-At least one receiver must be enabled per [pipeline](../docs/pipelines.md) to be a
-valid configuration.
```
Comment (tigrannajaryan) [inline receiver/README.md:55]: I think this is important to keep.

File: receiver/README.md [hunk 5]
```diff
@@ -46,10 +55,27 @@ receivers:
...
+must be unique.
+
+For the example above:
+
 - Receiver 1 has full name `examplereceiver`.
 - Receiver 2 has full name `examplereceiver/settings`.
 
-All receivers expose a setting to disable it, by default receivers are enabled.
-At least one receiver must be enabled per [pipeline](../docs/pipelines.md) to be a
-valid configuration.
```
Comment (flands) [inline receiver/README.md:55]: > At least one receiver must be enabled per pipeline to be a valid configuration.

File: receiver/README.md [hunk 6]
```diff
@@ -46,10 +55,27 @@ receivers:
 A receiver instance is referenced by its full name in other parts of the config,
 such as in pipelines. A full name consists of the receiver type, '/' and the
 name appended to the receiver type in the configuration. All receiver full names
-must be unique. For the example above:
+must be unique.
+
+For the example above:
+
 - Receiver 1 has full name `examplereceiver`.
 - Receiver 2 has full name `examplereceiver/settings`.
```
Comment (flands) [inline receiver/README.md:64]:
```suggestion

> At least one receiver must be enabled per pipeline to be a valid configuration.
```

**Comment by bogdandrutu** [comment]
All 3 PRs failed the check links, can you take a look?

**Review by tigrannajaryan — APPROVED** [review]

**Pull request merged**